src/main/java/com/launchdarkly/client/LDUser.java

```diff
@@ -107,11 +116,14 @@ JsonPrimitive getAnonymous() {
   }
 
   JsonElement getCustom(String key) {
-    return custom.get(key);
+    if (custom != null) {
```

Comment:
This became a problem when running the restwrapper which deserializes user objects from json, so the custom field doesn't automatically get created as an empty map. A user of this library could do the same, so I think leaving it as-is is safer.